lighthouse-cli/bin.ts

```diff
@@ -229,11 +235,32 @@ function lighthouseRun(addresses: Array<string>, config: Object, flags: Object)
   return lighthouse(address, flags, config)
     .then((results: Results) => Printer.write(results, outputMode, outputPath))
     .then((results: Results) => {
+      const filename = `${assetSaver.getFilenamePrefix({url: address})}.report.html`;
+
       if (outputMode === Printer.OutputMode[Printer.OutputMode.pretty]) {
-        const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
         Printer.write(results, 'html', filename);
       }
 
+      // Generate report.html, host it and open it in the default browser
+      if (flags.view) {
+        performanceXServer.startServer(0).then((port: number) => {
+          const filePath = `${performanceXServer.FOLDERS.REPORTS}/${filename}`;
+          const htmlExist = outputMode === Printer.OutputMode[Printer.OutputMode.pretty]
+            || (outputPath !== 'stdout' && outputMode === Printer.OutputMode[Printer.OutputMode.html]);
```

Comment:
for some conflicting advice here: I'm still not a fan of the fragility of this check. It may not be worth reusing the `pretty` report...I'd personally prefer just re-printing (~100ms for a decent report) and longer term we could do some kind of refactor like separating generation of the report html from saving to file

@paulirish for his opinion